phpBB/includes/functions.php

```diff
@@ -3935,6 +3935,7 @@ function page_header($page_title = '', $display_online_list = false, $item_id =
 		'CURRENT_USER_AVATAR'			=> phpbb_get_user_avatar($user->data),
 		'CURRENT_USERNAME_SIMPLE'		=> get_username_string('no_profile', $user->data['user_id'], $user->data['username'], $user->data['user_colour']),
 		'CURRENT_USERNAME_FULL'			=> get_username_string('full', $user->data['user_id'], $user->data['username'], $user->data['user_colour']),
+		'CURRENT_USER_GROUP_COLOR'		=> ' color: #' . $user->data['user_colour'],
```

Comment:
Not a fan of having CSS syntax in here, I'd rather have navbar_header.html adjusted to:
`{{ Icon('iconify', 'fa:caret-down', '', true, '', {'style': 'color: #' ~ CURRENT_USER_GROUP_COLOR}) }}`